ray-operator/config/samples/pytorch-resnet-image-classifier/ray-job.pytorch-image-classifier.yaml

```diff
@@ -31,7 +31,7 @@ spec:
           serviceAccountName: pytorch-distributed-training
           containers:
             - name: ray-head
-              image: rayproject/ray:2.9.0
+              image: rayproject/ray:2.34.0
```

Comment:
This example no longer works because head node is on 2.34 and worker node is on 2.9 using ray-ml image. Should we revert this back to 2.9 or update the worker image to use the deprecated ray-ml images for now?